### PR TITLE
[onert/test] TFlite model test support all input type

### PR DESCRIPTION
**Pull request by hseok-oh**
- Support all input type
- Implement comparison function for all data type (copy from GenModelTest.h)
- Support thread setting environment variable

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

---

Part of #5212

### CODE REVIEW

File: tests/tools/tflite_loader/src/tflite_loader.cc [hunk 1]
```diff
@@ -45,7 +46,7 @@ const float DIFFERENCE_THRESHOLD = 10e-5;
   }
 
 // Read vector of floats from selected file
-std::vector<float> readData(const string &path)
+void readData(const string &path, uint8_t *dest)
```
Comment (seanshpark) [inline tests/tools/tflite_loader/src/tflite_loader.cc:49]: (optional) maybe add some assert checkes for `dest` ? caller can call with `nullptr`
also for below codes

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by seanshpark** [comment]
I think there should be another reviewer as I don't have full knowledge of the codes :)

**Comment by hseok-oh** [comment]
@glistening PTAL

**Comment by hseok-oh** [comment]
PTAL

**Review by seanshpark — APPROVED** [review]
LGTM

**Comment by hseok-oh** [comment]
I've updated to resolve conflict.

File: tests/tools/tflite_loader/src/tflite_loader.cc [hunk 2]
```diff
@@ -264,31 +318,69 @@ int main(const int argc, char **argv)
...
+        break;
+      case NNFW_TYPE_TENSOR_QUANT8_ASYMM_SIGNED:
+        matched = compareBuffersExact<int8_t>(ref_output.int8, output, out_idx);
+        break;
+      case NNFW_TYPE_TENSOR_INT32:
+        matched = compareBuffersExact<int32_t>(ref_output.i32, output, out_idx);
+        break;
+      case NNFW_TYPE_TENSOR_FLOAT32:
+        // TODO better way for handling FP error?
+        for (uint32_t e = 0; e < num_elems(&ti); e++)
```
Comment (glistening) [inline tests/tools/tflite_loader/src/tflite_loader.cc:352]: @hseok-oh (optional) We may compare in the same way of `tflite_run`:

```c++
auto equals = [tolerance](float lhs, float rhs) {
  // NOTE Hybrid approach
  // TODO Allow users to set tolerance for absolute_epsilon_equal
  if (nnfw::misc::fp32::absolute_epsilon_equal(lhs, rhs))
  {
    return true;
  }

  return nnfw::misc::fp32::epsilon_equal(lhs, rhs, tolerance);
};
```

I am not saying `tfflite_run`'s way is best, but would like to use same method.

**Review by glistening — APPROVED** [review]
LGTM

File: tests/tools/tflite_loader/src/tflite_loader.cc (hunk 2 above)
Comment (hseok-oh) [reply tests/tools/tflite_loader/src/tflite_loader.cc:352]: Thanks. I'll try that later.

**Pull request merged**